Cloud Application Manager/Core Concepts/context-switcher.md

```diff
@@ -0,0 +1,119 @@
+* [Creating a new scope](#creating-a-new-scope)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+This article is meant to give a definition of Cloud Application Manager’s Context Switcher and an explanation of its features. Before reading this article, a previous knowledge of [organizations, cost centers and workspaces](workspaces-and-collaboration.md) is recommended. 
+
+### Audience
+
+Cloud Application Manager users.
+
+### Prerequisites
+
+An active Cloud Application Manager account.
+
+### Introduction
+
+Context Switcher is a tool located within the Global Navigation bar of Cloud Application Manager, and allows users changing between different scopes they have access to, be they Organizations, Cost Centers or Workspaces, so they can use and create different resources depending on the scope they have selected.
+
+![Context Switcher - General View](../../images/cloud-application-manager/context-switcher.png)
+
+It provides at a glance all the necessary information so that users know in which scope they are situated at every moment. Thus, this component offers the following data when collapsed:
+
+* Kind of Scope (in blue)
+
+* Scope’s icon/avatar (in red)
+
+* Scope’s hierarchy (in green)
+
+![Context Switcher - In Detail](../../images/cloud-application-manager/context-switcher-detail.png)
+
+### Sections
+
+Clicking on the context switcher will create an overlay with extended information and functionality, and divided into different sections, which correspond to scope types (Organizations, Cost Centers and Workspaces), apart from a Favorites section:
+
+![Context Switcher - Main View](../../images/cloud-application-manager/context-switcher-general.png)
+
+#### Search bar
+
+This will let users search for a concrete scope they have access to. It is important to highlight that results will be different depending on which kind of scope the user has selected, so it won’t be possible to seek for a scope not belonging to the active section so, for example, when the Organization section is selected, only organizations are searched.
+
+![Context Switcher - Search Bar](../../images/cloud-application-manager/context-switcher-search-bar.png)
+
+#### Selection menu
+
+There are several selectable options here depending on user's permissions (organization administrators can see Organizations and Cost Centers options while cost center administrators could only see their associated Cost Centers), and apart from the different kinds of scopes (Organizations, Cost Centers and Workspaces), there is also a Favorites option, which allows to store together those scopes marked as favorites by the user. 
+
+![Context Switcher - Selection Menu](../../images/cloud-application-manager/context-switcher-selection-menu.png)
+
+#### Main view
+
+When an option is selected, main view displays all available scopes of selected kind for a given user, and depending on that kind of scope some differences arise:
```

Comment:
```suggestion
When a scope type is selected, the main view displays all available scopes of the selected kind for a given user, and depending on that kind of scope some differences arise:
```
In general, I don't like it too much when you say scope `option`. I'd rather use `type`, `kind` or even `tab`. Please, review usage in all the doc.